src/App.css

```
@@ -1,42 +1,47 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media ( prefers-reduced-motion : no-preference ) {
-
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-
+.main-page {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: nowrap;
 }
 
-.App-header {
-  align-items: center;
-  background-color: #282c34;
-  color: white;
+.categories-list {
   display: flex;
-  flex-direction: column;
-  font-size: calc(10px + 2vmin);
+  flex-wrap: wrap;
+  border: 1px solid grey;
+  border-radius: 1%;
+  background-color: lightgrey;
+  display: flex;
+  flex-wrap: wrap;
   justify-content: center;
-  min-height: 100vh;
+  width: 30vw;
+  height: 100vh;
 }
 
-.App-link {
-  color: #61dafb;
+.category {
+  font-size: 20px;
+  text-align: center;
+  background-color: rgba(0, 0, 255, 0.466);
+  border: 1px solid gray;
+  margin: 10px;
 }
 
-@keyframes App-logo-spin {
-
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+.product-list {
+  border: 1px solid grey;
+  border-radius: 1%;
+  background-color: yellow;
+  width: 70vw;
+  height: 100vh;
+}
+.product-list h1 {
+  text-align: center;
+}
```

O que acha de acrescenter uma linha em branco entre esses blocos?
```suggestion
}

```